django/db/backends/oracle/features.py

```diff
@@ -120,6 +120,9 @@ class DatabaseFeatures(BaseDatabaseFeatures):
             "migrations.test_operations.OperationTests."
             "test_alter_field_pk_fk_db_collation",
         },
+        "Oracle cannot cast NCLOB column to NUMBER.": {
```

Comment:
We don't cast on Oracle:
```suggestion
        "Oracle doesn't support comparing NCLOB to NUMBER.": {
```